tests/test-td-payload/src/testtdinfo.rs

```diff
@@ -27,6 +27,7 @@ pub struct TdInfoRetData {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Tdinfo {
     pub name: String,
+    pub hob: usize,
```

Comment:
hob is runtime data. Why mix it with static data?

Reply:
hob is not needed in Tdinfo. It will be removed.